doc/source/theory/unresolved_cfd-dem/unresolved_cfd-dem.rst

```diff
@@ -120,26 +146,43 @@ where :math:`n_p` is the number of particles with centroid inside the cell :math
 
-where :math:`\varepsilon_{f,i}` is the void fraction calculated by PCM and :math:`\varepsilon_{f,j}` the projected void fraction.
+.. image:: images/spm.png
+   :width: 49% 
+   :align: center
+   
+The void fraction in a cell using SPM can be written as: 
 
-Then, we assemble and solve the following:
+.. math:: 
+    \varepsilon_f = 1 - \frac{\sum_{i}^{n_p}\sum_{i}^{n_{sp}} V_{sp,j}}{V_\Omega}
 
-.. math::
-    \int_{\Omega} \varphi_i \varepsilon_{f,j} \varphi_j d \Omega = \int_{\Omega} \varepsilon_{f,i} \varphi_i d \Omega
+where :math:`n_{sp}` is the number of pseudo-particles j belonging to particle i with centroid inside the cell :math:`\Omega` with volume :math:`V_{\Omega}` and :math:`V_{sp}` is the volume of the satellite point. The satellite point method suffers from the same limitations as the PCM. However, it is slightly less discontinuous due to the refined nature of the particles.
 
+The Quadrature Centered Method
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It is an analytical method which decouples the averaging volume from the mesh cells. It constructs an averaging sphere centered at each quadrature point in a given cell, and it calculates the void fraction directly in the averaging volume at the quadrature point. Since the sphere-sphere (particle-averaging sphere) intersection is analytically easier to calculate than sphere-polyhedron (particle-mesh cell), this method is less expensive than other analytical methods as the intersection does not involve the calculation of trigonometric functions at each CFD time step. The advantage of this method is that the void fraction varies within a cell. Additionally, particles in neighboring cells can affect the void fraction of the current cell. This allows the method to be continuous in both space and time. This is advantageous specially in solid-liquid systems where the term :math:`\rho_f \frac{\partial \epsilon_f}{\partial t}` of the continuity equation is very stiff and unstable when there exist even small discontinuities in the void fraction and where it explodes when :math:`\Delta t_{CFD} \to 0`. 
 
-Lethe also has the option of smoothing the void fraction profile, which helps to mitigate sharp discontinuities. To do so, the left hand side of the previous equation is substituted by:
+An averaging volume sphere is constructed around each quadrature point. All particles lying in the sphere will contribute to the void fraction value of this quadrature point. Therefore, a cell will be affected by the particles lying in it and in its neighboring cells.
 
-.. math::
-    \iint_\Omega L^2 \nabla \phi_i \nabla \phi_j d\Omega
+.. image:: images/qcm.png
```

Comment:
Nice image! I would suggest you add a white background only because it is not possible to see it properly in dark mode. 
![image](https://user-images.githubusercontent.com/65647639/222508629-e3a975ed-11a1-454d-a75b-3d1485cc58b0.png)
![image](https://user-images.githubusercontent.com/65647639/222508706-fc2c9257-53db-4292-9ce1-b19ab2336895.png)

Reply:
Oh you are right. The background has been changed.